nixops/modules/rabbitmq/default.nix

```diff
@@ -114,6 +54,12 @@ in {
       };
       environment = {
         PUBLISH_PORT = "9419";
+        RABBIT_URL = "https://${cfg.domain}";
+        SKIPVERIFY = "true"; # yeah, sue me. the current version in nixpkgs
+        # fails on https://github.com/kbudde/rabbitmq_exporter/issues/149
+        # and while I'd like to upgrade it, it was a can of worms... and
+        # this username and password isn't very interesting, anyway.
+
         RABBIT_CAPABILITIE = "bert,no_sort";
```

Comment:
Not in this diff, but, uh, should this be `RABBIT_CAPABILITIES`?

Reply:
lol, I wonder what the story is here. Let's delete it.

```suggestion
```